activitysim/core/test/test_los.py

```diff
@@ -166,9 +166,9 @@ def test_30_minute_windows():
     assert network_los.skim_time_period_label(36) == 'PM'
     assert network_los.skim_time_period_label(46) == 'EV'
 
-    pd.testing.assert_series_equal(
+    np.testing.assert_array_equal(
```

Comment:
Why switch from pd to np testing classes? Performance?

Reply:
The `skim_time_period_label()` method now returns an array as seen [here](https://github.com/ActivitySim/activitysim/pull/481/files/78763b6598c090889735a087239765df9b0963f3#diff-7c1c031529711ac262c8a56155be3b10a61a3f906b6c1ca38f5efcd927a5e499R580). Going to update the docstring for this method now to reflect the change.